go/runtime/host/sandbox/sandbox.go

```diff
@@ -209,18 +210,10 @@ func (r *sandboxedRuntime) WatchEvents(context.Context) (<-chan *host.Event, pub
 }
 
 // Implements host.Runtime.
-func (r *sandboxedRuntime) Start() error {
-	r.Lock()
-	defer r.Unlock()
-
-	if r.started {
-		return nil
-	}
-	r.started = true
-
-	go r.manager()
-
-	return nil
+func (r *sandboxedRuntime) Start() {
+	r.startOnce.Do(func() {
```

Comment:
I only added this because I was refactoring the `Start` method to not return an error. I agree that we can refactor this more if needed later.

I removed `quitCh` for now which was indeed unused.